r/R/dplyr-functions.R

```diff
@@ -893,6 +901,12 @@ output_type <- function(fun, input_type) {
     input_type
   } else if (fun %in% c("mean", "stddev", "variance")) {
     float64()
+  } else if (fun %in% "tdigest") {
+    if (hash) {
+      fixed_size_list_of(float64(), 1L)
+    } else {
+      float64()
```

Comment:
I'm not sure what tie-breaking algorithm is being used, but wouldn't median/quantile be expected to return a value of the same type as the data?

```suggestion
      fixed_size_list_of(input_type, 1L)
    } else {
      input_type
```

Reply:
No, currently they only accept numeric input and they they always return `float64()`. I believe this is because of tie-breaking
```r
call_function("tdigest", Array$create(1:4), options = list(q = 0.5))
## Array
## <double>
## [
##   2.5
## ]
```